src/cli/Clip.cpp

```diff
@@ -70,28 +77,39 @@ int Clip::executeWithDatabase(QSharedPointer<Database> database, QSharedPointer<
         return EXIT_FAILURE;
     }
 
+    QString selectedAttribute = parser->value(AttributeOption);
     QString value;
-    if (clipTotp) {
+    bool found = false;
+    if (parser->isSet(TotpOption) || selectedAttribute == "totp") {
         if (!entry->hasTotp()) {
             errorTextStream << QObject::tr("Entry with path %1 has no TOTP set up.").arg(entryPath) << endl;
             return EXIT_FAILURE;
         }
 
+        found = true;
         value = entry->totp();
     } else {
-        value = entry->password();
+        for (const QString& key : entry->attributes()->keys()) {
+            if (key.compare(selectedAttribute, Qt::CaseSensitivity::CaseInsensitive) == 0) {
+                value = entry->attribute(key);
```

Comment:
@sjamesr why not use `value = entry->attributes()->value(selectedAttribute)`?

Reply:
I wanted case insensitivity, what do you think?

Reply:
I just realized that means the attribute name is case sensitive when we use the `-a` option in the `show` CLI command :grimacing: 

Maybe we could make it case insensitive there too and extract that logic in a function. Not sure `EntryAttribute::` is the right place though, maybe in `cli/Utils`?

Reply:
I extracted this code to Utils::findAttributes and updated `clip` and `show` to use it. If the user-specified attribute name is ambiguous, an error message is displayed by both `clip` and `show`.